### ISSUE TITLE
Suggestion to change Jade to WTHIT

### ISSUE THREAD

**Issue by Crasben** [Laskyyy/Create-Astral#168]
https://www.curseforge.com/minecraft/mc-mods/wthit

Megane it's the addon for WTHIT to make it show Create and other mods info
https://www.curseforge.com/minecraft/mc-mods/megane

**Comment by Laskyyy** [comment]
We used to use WTHIT before jade, funnily enough. I had a few recommends to switch to Jade which led to this decision, i'd be interested in the pros and cons of each.

**Comment by Crasben** [comment]
![image](https://github.com/Laskyyy/Create-Astral/assets/38433553/52dd0f8a-f95d-4ea8-8bb0-7ee9940da42b)
The image show that Jade (The left one) doesn't show what's in the inventory of create blocks, WTHIT (the one on the right) it shows what's in it. 
![image](https://github.com/Laskyyy/Create-Astral/assets/38433553/f998c838-3f98-450c-96e7-34b666849354)
The same with the tanks.
![image](https://github.com/Laskyyy/Create-Astral/assets/38433553/37488899-62e5-4207-8c05-6c95eb8d3430)
The same with Reborn machines.

WTHIT doesn't show if you can mine it or not, but Jade does.

**Comment by Crasben** [comment]
> But Jade also has one, [Jade Addons](https://www.curseforge.com/minecraft/mc-mods/jade-addons-fabric).
> 
> I think that make more sense add the jade addons instead of changing to WTHIT.

The modpack comes with that mod and i looked up the settings of Jade and the addon and i found that it couldn't show what WTHIT shows.

**Comment by The-Shortman** [comment]
I don't believe Jade will be replaced any time soon considering how long this issue has been open
